src/types/font.test.ts

```diff
@@ -21,4 +21,9 @@ describe("Font", (): void => {
       fontStyle: "italic"
     });
   });
+
+  it("size is left out of CSSProperties when not input", (): void => {
```

Comment:
The comment says size but the test says style...?

Reply:
Good catch